tests/testthat/test-print.R

```diff
@@ -62,3 +62,20 @@ test_that("Special characters are not re-encoded", {
     }
   )
 })
+
+
+test_that("save_html() language parameter is set", {
+  output <- tempfile(fileext = ".html")
+  # test for default
+  save_html("<h2>Howdy</h2>", output)
+  output_read <- readLines(output)
+  expect_true(
+    grepl("<html lang=\"en\">", paste(output_read, collapse = " "))
+  )
+  # test for fr
+  save_html("<h2>Howdy</h2>", output, lang = "fr")
+  output_read <- readLines(output)
+  expect_true(
+    grepl("<html lang=\"fr\">", paste(output_read, collapse = " "))
+  )
```

Comment:
Let's also make sure `lang` can be overridden via `<body>` (in case the end user doesn't have control over `save_html()`) 

```suggestion
  )
  save_html('<body lang="fr">Howdy</body>', output)
  output_read <- readLines(output)
  expect_true(
    grepl("<body lang=\"fr\">", paste(output_read, collapse = " "))
  )
```

Reply:
Now that I think about it, I don't think we need to hoist a body tag's `lang` attribute -- if the user does something like `save_html('<body lang="fr">Howdy</body>', output)`, then my understanding is that the browser will interpret everything in the body to be in French. The same is true for any top-level tag: the content inside will be interpreted using the specified `lang`.